docs/data/tree-view/rich-tree-view/customization/IconExpansionTreeView.js

```diff
@@ -1,8 +1,6 @@
 import * as React from 'react';
```

Comment:
WDYT about moving this demo in the *Expansion* page?

And maybe we can find another "Common example" in the near future to avoid having just one.

Reply:
Forget about the other example, we have the 3D example from Jose :+1: